readline.go

```diff
@@ -55,6 +55,9 @@ type Config struct {
 	EnableMask bool
 	MaskRune   rune
 
+	// Whether to maintain an undo buffer (Ctrl+_ to undo if enabled)
+	Undo bool
```

Comment:
Possibly enable by default?

Reply:
I'm not sure how much demand there is for this as a feature; for example, I can't find an open or closed issue requesting it in the upstream repository. My end goal here is to get `fq` to migrate to the fork, not to provide undo for its own sake :-)

What's your thinking on making it the default? If the rationale is that there is significant demand for an undo feature, I'm not sure. If it's that as a non-default, it's at risk of becoming a poorly maintained "dark corner" of the library, then that makes more sense to me, but I think I still have a mild preference for leaving it off by default.

Reply:
I see, mostly my thinking is to default enabled what most readliney things like bash, zsh etc enabled by default. But i don't have that strong opinion about is really :) but yeah i agree about maintainability, figure out some way of adding automated tests for things like this might be the solution? btw i noticed this thread the other day https://lobste.rs/s/qfbqsj/has_anyone_found_robust_way_test the comment about using tmux is a bit interesting.

And just to be clear, i'm totally fine with explicitly enable it for fq